packages/terra-paginator/src/Paginator.scss

```diff
@@ -0,0 +1,112 @@
+@import '~terra-mixins';
+
+:local {
+  .paginator {
+    align-items: center;
+    display: flex;
+    justify-content: center;
+
+    &.pageless {
+      justify-content: space-between;
+    }
+
+    &.progressive {
+      justify-content: space-between;
+    }
+
+    .nav-link {
+      background: var(--terra-paginator-nav-link-background);
+      border: var(--terra-paginator-nav-link-border);
+      border-radius: var(--terra-paginator-nav-link-border-radius, 3px);
+      color: var(--terra-paginator-nav-link-foreground-color, #0059a8);
+      cursor: pointer;
+      display: inline-block;
+      line-height: var(--terra-paginator-nav-link-line-height, 1.43);
+      margin-left: var(--terra-paginator-nav-link-margin-left, 0.14285rem);
+      margin-right: var(--terra-paginator-nav-link-margin-right, 0.14285rem);
+      min-width: var(--terra-paginator-nav-link-min-width, 2rem); // Helps ensure limited resizing as you page through large numbers.
+      padding: var(--terra-paginator-nav-link-padding, 3px);
```

Comment:
Size adjustment: `3px 3px` to `0.35714rem 0.5rem`

Reply:
https://github.com/cerner/terra-core/pull/1445/commits/37187bdacdbd135aef8fab5bf1ca8b24cbbc2bf9